### DISCUSSION TITLE
Questions regarding installation and building

### DISCUSSION THREAD

**Discussion by F-36** [cyring/CoreFreq#438] [Q&A] [answered]
1. Is systemd-boot also going to work?

2. Mandatory and optional linux kernel header files to build modules - I have `CONFIG_X86_MSR=m`, should I do `modprobe msr` beforehand?
 
Thanks

**Comment by cyring** [comment, marked as answer]
Hello, 

> 1. Is systemd-boot also going to work?

It doesn't matter for _CoreFreq_ t'il you have correctly built against current kernel and have a terminal to load driver then run Damon, next Client. 

> 2. Mandatory and optional linux kernel header files to build modules - I have `CONFIG_X86_MSR=m`, should I do `modprobe msr` beforehand?

From `CONFIG_X86_MSR`, I'm just using some MSR addresses definitions during driver build. 
During **runtime**, _CoreFreq_ doesn't need the msr tools. 
Thus you **don't** have to `modprobe msr`

> 
> Thanks

Enjoy _CoreFreq_
